yaml-tests/src/main/java/com/apple/foundationdb/relational/yamltests/block/TestBlock.java

```diff
@@ -436,6 +446,7 @@ private void executeInParallelizedMode(Collection<Consumer<RelationalConnection>
         }
         // Iterate through the futures to catch any uncaught errors/exceptions from the submitted tasks.
         for (var future : futures) {
+            Verify.verify(!future.isCancelled());
```

Comment:
What's the point of this `Verify`? Were there canceled futures sneaking in here causing trouble?

Reply:
I think there is a bug but I cannot prove it. So if you submit executables more of less in bulk like we do a couple of lines up, there is a chance that an executable is still in the queue of the executor pool when the pool is shut down. It will not wait for those tasks to start and finish but just cancel those futures. I cannot repro it so I left that in here.

Reply:
I have no idea why we are not using a countdown latch or similar here.